src/ngx_http_lua_directive.c

```diff
@@ -446,10 +446,12 @@ ngx_http_lua_rewrite_by_lua_block(ngx_conf_t *cf, ngx_command_t *cmd,
 char *
 ngx_http_lua_rewrite_by_lua(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 {
-    u_char                      *p, *chunkname;
-    ngx_str_t                   *value;
-    ngx_http_lua_main_conf_t    *lmcf;
-    ngx_http_lua_loc_conf_t     *llcf = conf;
+    u_char                          *p, *chunkname;
+    ngx_str_t                       *value;
+    ngx_http_lua_main_conf_t        *lmcf;
+    ngx_http_lua_loc_conf_t         *llcf = conf;
+    ngx_array_t                     *rewrite_handlers = llcf->rewrite_handlers;
+    ngx_http_lua_phase_handler_t    *phase_handler;
```

Comment:
Local C variables sholud use short names (Huffman principle). How about use `ph` for `phase_handler` here?